charts/super-agent/values.yaml

```diff
@@ -138,7 +138,7 @@ super-agent-deployment:
         #   endpoint: https://opamp.service.newrelic.com/v1/opamp
         #   headers:
         #     # TODO this value should be automatically injected
-        #     api-key: LICENSE_KEY
+        #     api-key: ${nr-env:NR_LICENSE_KEY}
```

Comment:
I think this is wrong

Reply:
nice point 😕 I guess that also Juanma PR has the same issue of "enabling by default" opamp

Reply:
I am checking how complex it is to include enable/disable for opamp